.github/workflows/release_to_azure.yml

```diff
@@ -53,9 +53,11 @@ jobs:
         uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938
 
       - name: Build backend
+        if: ${{ needs.wf_vars.outputs.enable_builds == 'true' }}
         uses: ./.github/actions/build-backend
         with:
-          version: ${{ needs.pre_job.outputs.version }}
+          version: ${{ needs.wf_vars.outputs.version }}
```

Comment:
Is there a reason we included this change? The default of build-backend is to skip validation, so why are we passing true here? Ditto for the release_to_github change.